content/en/real_user_monitoring/browser/setup/client.md

```diff
@@ -1982,7 +1982,21 @@ window.DD_RUM.init({
 {{% /tab %}}
 {{% tab "Error Tracking" %}}
 
-TypeScript is not supported for Error Tracking.
+
+Types are compatible with TypeScript >= 3.8.2. For earlier versions, import JavaScript sources and use global variables to avoid any compilation issues:
+
+```javascript
+import '@datadog/browser-rum/bundle/datadog-rum'
+
+window.DD_RUM.init({
+  applicationId: 'XXX',
+  clientToken: 'XXX',
+  site: 'datadoghq.com',
+  trackUserInteractions: true,
+   trackResources: true,
+  ...
+})
+```
```

Comment:
~~~suggestion
Types are compatible with TypeScript >= 3.8.2. To initialize the SDK, use the following code snippet.

<div class="alert alert-info"><strong>Note</strong>: For earlier versions of TypeScript, import JavaScript sources and use global variables to avoid any compilation issues.</div>

```javascript
import '@datadog/browser-rum/bundle/datadog-rum'

window.DD_RUM.init({
  applicationId: 'XXX',
  clientToken: 'XXX',
  site: 'datadoghq.com',
  trackUserInteractions: true,
   trackResources: true,
  ...
})
```
~~~